skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -269,6 +269,33 @@ def test_binmap3d():
     rbinmap2 = binstat2.binmap
 
     assert_array_almost_equal(rbinmap1[0][::1000], rbinmap2[0][::1000])
-    assert_array_almost_equal(rbinmap1[0][::1000], np.array([1, 10,  9,  8, 7,
-                                                             6,  5,  4,  3, 2,
+    assert_array_almost_equal(rbinmap1[0][::1000], np.array([1, 10, 9, 8, 7,
+                                                             6, 5, 4, 3, 2,
                                                              1]))
+
+
+@pytest.mark.parametrize(['stat', 'stat_f'], stats_list)
+def test_BinnedStatistics1D_mask(stat, stat_f):
```

Comment:
Can you add a short description about what the test is testing (like a one-line test function docstring)

Reply:
I also agree with docstrings on tests :-), i think they're lacking!